flows/interfaces.go

```diff
@@ -218,6 +218,10 @@ type LogEntry interface {
 	Event() Event
 }
 
+type FlowStack interface {
+	HasFlow(FlowUUID) bool
```

Comment:
Should we just add a method on session? Then we could get rid of this interface entirely?